app/src/router/routes.js

```diff
@@ -361,6 +361,15 @@ const routes = [
       breadcrumb: ['tool-list', 'tool-webadmin']
     }
   },
+  {
+    name: 'tool-settings',
+    path: '/tools/settings',
+    component: () => import(/* webpackChunkName: "views/tools/settings" */ '@/views/tool/ToolSettings'),
+    meta: {
+      args: { trad: 'tools_yunohost_settings' },
+      breadcrumb: ['tool-list', 'tool-settings']
+    }
```

Comment:
Yo ! Sorry another change on config panels has been merged. It adds routing to tabs.
It should only require that you change your route definition to enable it.
Tabs should then be accessible on `/tools/settings/security` or `/tools/settings/other` in your case.

```suggestion
    path: '/tools/settings',
    component: () => import(/* webpackChunkName: "views/tools/settings" */ '@/views/tool/ToolSettings'),
    children: [
      {
        name: 'tool-settings',
        path: ':tabId?',
        component: () => import(/* webpackChunkName: "components/configPanel" */ '@/components/ConfigPanel'),
        props: true,
        meta: {
          routerParams: [],
          args: { trad: 'tools_yunohost_settings' },
          breadcrumb: ['tool-list', 'tool-settings']
        }
      }
    ]
```